tests/test_diff.py

```diff
@@ -36,7 +36,7 @@ def test_diff_points(
     repo_version, output_format, data_working_copy, geopackage, cli_runner
 ):
     """ diff the working copy against HEAD """
-    data_archive = "points2" if repo_version == 2 else "points"
+    data_archive = "points" if repo_version == 2 else "points"
```

Comment:
should remove the V1_OR_V2 parametrization on this test

Reply:
Done